packages/datetime/src/_daterangepicker.scss

```diff
@@ -4,26 +4,15 @@
 @import "./common";
 
 // react-day-picker does not conform to our naming scheme
-
 /* stylelint-disable selector-class-pattern */
+
 .#{$ns}-daterangepicker {
   display: flex;
-  white-space: nowrap;
```

Comment:
This style has been around since the beginning of this OSS repo, and it's not clear exactly what the benefit is. It's a strange style to inherit for anyone specifying a custom `footerElement`, so I've chosen to remove it in this PR. I'll keep an eye out for unintended breaks caused by this.